src/components/Item.jsx

```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Link } from "react-router-dom";
 
-const Item = ({ movie }) => {
-  const genres = movie.genres.join(" & ");
+const Item = (props) => {
+
+  const { id, poster_path, overview, title, release_date } = props.movie;
+  const genres = props.movie.genres.join(" & ");
 
   return (
     <div>
-      <img src={movie.poster_path} className="thumbnails" alt={movie.overview} />
+      <Link to={'/film/' + id}>
```

Comment:
You can use template string here as well.

Reply:
Template string used.